### DISCUSSION TITLE
To add TR2 swimming into TR1X

### DISCUSSION THREAD

**Discussion by LoneckCZ** [LostArtefacts/TRX#1376] [Ideas]
Can you please add TR2 swimming controls into TR1X? I mean if you press left or right while swimming it is quite sensitive in TR1. It is not that sensitive in TR2.

**Comment by aredfan** [comment]
Hey! We have an issue for your request in #1003.  It seems Lara has different turning speed while underwater in TR2.

Currently this feature is considered low priority meaning it may or may not be worked on in the future.

**Comment by LoneckCZ** [comment]
> Hey! We have an issue for your request in #1003. It seems Lara has different turning speed while underwater in TR2.
> 
> Currently this feature is considered low priority meaning it may or may not be worked on in the future.

Thanks for reply. I hope this will be implemented in the future.
